MAINTAIN.md

```diff
@@ -239,6 +239,34 @@ Some github labels are used to trigger certain jobs:
 * `needs:response` - close PR after a certain amount of time if author doesn't
   respond
 
+Vim patches
+-----------
+
+Multiple Vim versions are tracked in `version.c`, so that `has('patch-x.y.z')`
+works even if `v:version` is "behind". Whenever we "complete" merging all
+patches from a Vim `v:version`, the steps to bump `v:version` are:
+
+1. Remove the fully-merged version from `version.c:Versions`.
```

Comment:
Did you mean
```
char *Versions[] = { "8.1", "8.2", "9.0", "9.1" };
```
to remove `"8.1"`?
That's automated too.

Reply:
What steps are needed to remove a version?

Reply:
True.
Step 3 is required so that maintainer does not need a `version.c` MR to publish a release.

I am only saying that  `step 1` is redundant.
If `version.c` does need these manual edits, then vimpatch.lua missed edge cases during the refactor.
As long as vimpatch.lua can make Nvim jump to the correct lines and run the correct commands,
It should work on empty/incomplete arrays and generate the same result (assuming no merged vim-patch tokens like v8.2.0000).